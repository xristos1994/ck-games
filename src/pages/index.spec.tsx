import React from 'react';
import renderer from 'react-test-renderer';
import HomePage from './index';
import EL_HomePage from './el/index';
import EN_HomePage from './en/index';

const Home = () => {
  return (
    <div>
      Home
    </div>
  );
};

jest.mock('@page-components/', () => ({
  __esModule: true,
  Home: Home
}));

describe('pages/Home', () => {
  it('renders correctly', () => {
    let tree = renderer.create(
      <HomePage />
    ).toJSON();
    expect(tree).toMatchSnapshot();

    tree = renderer.create(
      <EL_HomePage />
    ).toJSON();
    expect(tree).toMatchSnapshot();

    tree = renderer.create(
      <EN_HomePage />
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
