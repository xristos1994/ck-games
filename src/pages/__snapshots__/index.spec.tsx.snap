// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`pages/Home renders correctly 1`] = `
<div>
  Home
</div>
`;

exports[`pages/Home renders correctly 2`] = `
<div>
  Home
</div>
`;

exports[`pages/Home renders correctly 3`] = `
<div>
  Home
</div>
`;
